types/aws-lambda/trigger/dynamodb-stream.d.ts

```diff
@@ -1,6 +1,6 @@
 import { Handler } from "../handler";
 
-export type DynamoDBStreamHandler = Handler<DynamoDBStreamEvent, void>;
+export type DynamoDBStreamHandler = Handler<DynamoDBStreamEvent, undefined | { state: any }>;
```

Comment:
```suggestion
export type DynamoDBStreamHandler = Handler<DynamoDBStreamEvent, undefined | { state: any }>;
```

type any is disregarded, for unknown maps use: `Map<String, String>` as suggested in the AWS docuemntation